app/controllers/backend/interventions_controller.rb

```diff
@@ -255,9 +255,17 @@ def new
       from_request = Intervention.find_by(id: params[:request_intervention_id])
       @intervention = from_request.initialize_record if from_request
 
+      @show_map = current_user.preference('interface.interventions.map_show', true)
+
       render(locals: { cancel_url: { action: :index }, with_continue: true })
     end
 
+    def update_map_preference
```

Comment:
This action can be more RESTful. Can you consider to create a dedicated controller for `Preference` with a strong validation of preferences which can be updated directly through PATCH requests.